lab_3/main.py

```diff
@@ -187,7 +296,18 @@ def create_from_tokens(self, encoded_corpus: tuple, ngram_sizes: tuple) -> int:
             (((1, 2), (2, 3), (3, 1)), ((1, 4), (4, 5), (5, 1)), ((1, 2), (2, 6), (6, 7), (7, 7), (7, 8), (8, 1))),
         )
         """
-        pass
+        for n in ngram_sizes:
+            summa = 0
+            trie = NGramTrie(n, self.storage)
+            if trie.extract_n_grams(encoded_corpus) or trie.get_n_grams_frequencies():
+                return 1
+            trie.extract_n_grams(encoded_corpus)
+            trie.get_n_grams_frequencies()
```

Comment:
If there is no error, you perform the same actions twice. Consider calling the methods only once and saving their return value for a later check.